conans/cli/output.py

```diff
@@ -107,54 +106,76 @@ def rewrite_line(self, line):
         self.stream.flush()
         self._color = tmp_color
 
-    def _write_message(self, msg, fg=None, bg=None):
+    def _write_message(self, msg, level_str, fg=None, bg=None):
         if conan_output_level == LEVEL_QUIET:
             return
-        tmp = ""
+
+        def json_encoder(_obj):
+            try:
+                return json.dumps(_obj)
+            except TypeError:
+                return repr(_obj)
+
+        if conan_output_logger_format:
+            import datetime
+            the_date = datetime.datetime.utcnow().replace(tzinfo=datetime.timezone.utc).isoformat()
+            ret = {"json": {"level": level_str, "time": the_date, "data": msg}}
```

Comment:
Why creating a json and dumping it instead of direct string format?

Reply:
Because I want a logger with json contents. I don't get the question.

Reply:
Yes, that was my question, if this was just a convenience for formatting or something, or it was really intended to be json output. Clear now, thanks.